src/components/agencySelection/AgencySelection.tsx

```diff
@@ -131,12 +121,11 @@ export const AgencySelection = (props: AgencySelectionProps) => {
 					.catch((error) => {
 						if (
 							error.message === FETCH_ERRORS.EMPTY &&
-							props.selectedConsultingType
+							props.consultingType.id
```

Comment:
Should this be:

```suggestion
							props.consultingType.id != null
```

? Or are we purposefully omitting the consulting type `0` here? That would mean we have hardcoded access to it though.

Reply:
Yes, please commit your suggestion :-)